src/std/rpc/json_req.rs

```diff
@@ -107,10 +107,18 @@ pub fn state_get_keys(key: StorageKey, at_block: Option<Hash>) -> Value {
     )
 }
 
+pub fn author_submit_extrinsic(xthex_prefixed: &str) -> Value {
+    author_submit_extrinsic_with_id(xthex_prefixed, REQUEST_TRANSFER)
+}
+
 pub fn author_submit_and_watch_extrinsic(xthex_prefixed: &str) -> Value {
     author_submit_and_watch_extrinsic_with_id(xthex_prefixed, REQUEST_TRANSFER)
 }
 
+pub fn author_submit_extrinsic_with_id(xthex_prefixed: &str, id: u32) -> Value {
```

Comment:
I don't get the ids, do you mind explaining it to me? :)

Reply:
I think it's just a Json RPC protocol thing. I copy pasted from what was already there :blush:

Reply:
Haha, same with me. :P